dev.yml

```diff
@@ -8,7 +8,7 @@ up:
   - custom:
       name: Minikube Cluster
       met?: test $(minikube status | grep Running | wc -l) -eq 2 && $(minikube status | grep -q 'Correctly Configured')
-      meet: minikube start --kubernetes-version=v1.10.0 --vm-driver=hyperkit
+      meet: minikube start --kubernetes-version=v1.11.1 --vm-driver=hyperkit
```

Comment:
Not officially supported by minikube yet, should we back it out?

Reply:
Yeah, I don't see another option (while we're still using Minikube, sigh).